cinema/views.py

```diff
@@ -0,0 +1,47 @@
+from django.shortcuts import get_object_or_404
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+class MovieCRUDView(APIView):
+    model = Movie
+    serializer_class = MovieSerializer
+
+    def get_object(self, pk: int):
+        instance = get_object_or_404(self.model, pk=pk)
+        return instance

```

Comment:
There is no need to allocate `instance` variable here, return the output directly from `get_object_or_404`.

Reply:
not fixed